src/utils/loadable.jsx

```diff
@@ -15,25 +15,15 @@ import CircularProgress from '@material-ui/core/CircularProgress';
     marginTop: 60,
     overflowX: 'auto',
   },
-  spinner: {
-    color: theme.palette.primary.main,
-  },
 }))
 class Loading extends PureComponent {
   content() {
-    const { classes, error, timedOut, pastDelay } = this.props;
+    const { error, timedOut, pastDelay } = this.props;
 
     if (error) {
       throw error;
     } else if (timedOut || pastDelay) {
-      return (
-        <CircularProgress
-          size={50}
-          classes={{
-            circleIndeterminate: classes.spinner,
-          }}
-        />
-      );
+      return <Spinner size={50} />;
```

Comment:
Remove `size` so we make sure the spinner looks identical to the one we use in the project view.